src/lusty/filesystem-explorer.rb

```diff
@@ -184,6 +188,31 @@ def all_files_at_view
       end
     end
 
+    def all_remote_files(view, view_str)
```

Comment:
I think it'd be okay to pass in `view` alone, and instantiate `view_str` within the function.  String creation is fast and simple function signatures are better.